test/Bond.spec.ts

```diff
@@ -0,0 +1,960 @@
+import { BigNumber, utils, BytesLike } from "ethers";
```

Comment:
We used to have something like this - https://github.com/porter-finance/v1-core/blob/4e45815c5f353331faaf07c356778755f7b7c5e5/test/end-to-end/normalFlow.spec.ts 

I removed it since it didn't seem to be testing anything we were not catching in our unit tests and was adding overhead when refactoring tests. We will look into re-adding something similar to the rari example.